docs/UserGuide.md

```diff
@@ -174,33 +175,37 @@ When total spending exceed the budget, UniSave will remind you to set a new budg
 Format: `set-b BUDGET`
 
 Example:
-`set-b 500`: Set the budget to 500.
+* `set-b 500`: Set the budget to 500.
 
 ![setBudget](images/setBudgetCommand.png)
 
 **Note** :
-1. You can change the Currency before setting budget with command <b>exchange cc/CURRENCY_CODE</b>, so that you can set your budget in another currency.
-to see a full list of supported currencies with their currency codes use command <b>show-codes</b>.
+1. You can change the Currency before setting budget with command `exchange cc/CURRENCY_CODE`, so that you can set your budget in another currency.
+to see a full list of supported currencies with their currency codes use command `show-codes`.
 2. Budget should only be positive numbers.
 
 Navigate back to the feature list: [Features](#3-features)
 
 #### 3.3 Add an expense: `add`
 
-Adds an expense.
+Save an new expense to UniSave.
```

Comment:
Why 'an' before new.
It might be better to just state: Add a new expense to UniSave, as the command is 'add'.